tasks/main.yml

```diff
@@ -27,7 +33,9 @@
 
 - name: Select NTP provider
   set_fact:
-    timesync_ntp_provider: "{{ timesync_ntp_provider_os_default if timesync_ntp_provider_current == '' else timesync_ntp_provider_current }}"
+    timesync_ntp_provider: "{{
+        timesync_ntp_provider_current|
+        default(timesync_ntp_provider_os_default, true)}}"
```

Comment:
please add space before the braces. https://github.com/oasis-roles/meta_standards/blob/master/README.md#jinja2-syntax

Reply:
Fixed.